.eslintrc.json

```diff
@@ -1,8 +1,16 @@
 {
   "plugins": [
-    "@cypress/dev"
+    "@cypress/dev",
+    "eslint-plugin"
   ],
   "extends": [
-    "plugin:@cypress/dev/general"
-  ]
+    "plugin:@cypress/dev/general",
+    "plugin:eslint-plugin/recommended"
+  ],
+  "rules": {
+    "eslint-plugin/prefer-message-ids": "off",
+    "eslint-plugin/require-meta-type": "off",
```

Comment:
Adding a type to a rule is a trivial one-line change so would be good to add types instead of disabling this.

https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/require-meta-type.md